Gaijinizer patch/Scripts.txt

```diff
@@ -2829,90 +2829,85 @@ It is already a limit
+But I didn't send this message just to congratulate you, I also wanted
+to tell you why you became the 'Intervener'...
+
+Firstly, the fact that you were storing magic while playing in order to
+reach that ending had to be kept under wraps. So that's why your world,
+the world farthest from the Demon Realm was chosen, a world where the
+very notion of 'magic' is nothing but fantasy.
+
+Oh but even though you were storing magic, it was funneled into that
+device you're using, so it won't affect your soul or anything like that.
+Having your soul emptied out just by playing a game would be some kind
+of bad joke.
+
+More importantly is the 'genre' of that device of yours...
+
+Why did you boot up that device? Naturally, everyone has their own
+various reasons for doing what they do, but maybe it's because you have
+some kind of 'preferred situation'?
+
+Even you aren't God, but an ordinary human. You, the intervener, needed the
+'aptitude' to stay with 'him' so he could manage to reach the ending.
+
+Someone who would actually enjoy being raped by a female demon, even perhaps
+a full-on masochist with a monster girl fetish. Someone who's open to non-human
+demons making an appearance.
+
+And above all, someone who can withstand the high difficulty, who can pick
+themselves back up no matter how many times it takes.
+
+If it were just a game, you could weaken the enemies and strengthen your
+characters, but that's not how it works with the 'device'. You can make some
+minor adjustments, but that's about it. What you did change however, was
+what's normally called 'Death's Destiny'. That's not something just any old fool
+can do.
+
+In order to identify those with your 'aptitude', a prototype of the intervention
+device was distributed called the 'Trial Version'. That was quite some time ago,
+do you remember what it contained? Do you remember where and how you intervened
+in that world? It was still an experimental device, so I imagine it was quite
+fragmented.
+
+...What? I'm already out of time? Okay, one last thing!
+This is easy to get confused, but the one who made this device and the one
+who made this game, are likely not the same person. The latter would've
+only thought they were making a niche porn game.
+
+We don't know who it was that put the device in there, but assuming they
+didn't just do it for a laugh, they probably reside in your world.
+Those guys at Hell's Gate only cooperated with them. Whoever it was, they
+wanted to change their own world by getting you to do something.
+
+But if 'his' journey has safely concluded, that might be a result of
+them laying the groundwork.
+
+Now then, putting all that aside, you've done some good work. You've
+successfully completed the game! Don't worry about reading too much
+into all this and just bask in your success.
+
+If we should ever meet again, I promise we'll get along just fine that
+time as well.
+ =================================================="#Ron
```

Comment:
I think you forgot to translate these last few lines, it looks exactly like the #MTLed lines, lol

Reply:
Okay, let's phrase it like this:
>_**If we ever meet again, I hope we'll be best friends.
That's a promise!**_